internal/config/artifact.go

```diff
@@ -27,7 +27,7 @@ func NewArtifactConfig(c *cli.Context) ArtifactConfig {
 
 func (c *ArtifactConfig) Init(args cli.Args, logger *zap.SugaredLogger) (err error) {
 	if c.Input == "" && args.Len() == 0 {
-		logger.Error(`trivy requires at least 1 argument or --input option`)
+		logger.Debug(`trivy requires at least 1 argument or --input option`)
 		return xerrors.New("arguments error")
```

Comment:
We should return `nil` here.

Reply:
If I return nil, the next step will be use the scanner. 

```
 ./trivy
2020-06-30T21:47:53.784-0300    FATAL   unable to initialize a scanner: unable to initialize a docker scanner: failed to parse the image name: could not parse reference:
```

Reply:
using trivy without parameters

![image](https://user-images.githubusercontent.com/8355621/86191754-3cb30680-bb1e-11ea-95f3-ec3d34676b44.png)

using trivy multi parameters, maybe It's interesting to do the same modifications.

![image](https://user-images.githubusercontent.com/8355621/86191918-a3d0bb00-bb1e-11ea-84b8-fcf481a253ea.png)

Reply:
Oh, sorry. I was confused 😄 .  We have two options.
- Use ShowAppHelpAndExit
- Define a custom error like `ErrNoTarget` and handles it.

We basically should not `exit` other than main.go or run.go, so I prefer the second one.

```
err = c.Init(true)
if err == artifact.ErrNoTarget {
	return nil
} else if err != nil {
	return xerrors.Errorf("failed to initialize options: %w", err)
}
```